packages/app/src/cli/models/extensions/ui-specifications/tax_calculations.ts

```diff
@@ -0,0 +1,24 @@
+import {createUIExtensionSpecification} from '../ui.js'
+import {BaseUIExtensionSchema} from '../schemas.js'
+import {schema} from '@shopify/cli-kit/node/schema'
+
+const TaxCalculationsSchema = BaseUIExtensionSchema.extend({
+  productionApiBaseUrl: schema.string(),
+  benchmarkApiBaseUrl: schema.string().optional(),
+  calculateTaxesApiEndpoint: schema.string(),
+})
+
+const spec = createUIExtensionSpecification({
+  identifier: 'tax_calculation',
```

Comment:
⚠️ the template folder needs to be exactly equal to this identifier.
you named it `tax_calculations`